comitup/nmmon.py

```diff
@@ -78,7 +80,29 @@ def cb_to(cb):
     timeout_add(1, cb_to, cb)
 
 
+def wired_changed_state(state, *args):
+    global wired_is_connected
+
+    log.info("MGAG: wired_changed_state {}".format(state))
+    from comitup.states import set_state
+
+    if state in PASS_STATES:
+        log.info("MGAG nmm - primary pass, set_state CONNECTED")
+        set_state('CONNECTED')
+        wired_is_connected = True
+        #send_cb(nm_dev_connect)
```

Comment:
This code is not "right".
Should not be calling `set_state()` directly, but should be going thru `send_cb()`.
However I wasn't able to wrap my head around doing that... I will need to try again.